counterpartylib/lib/config.py

```diff
@@ -88,7 +88,7 @@
 
 
 DEFAULT_REQUESTS_TIMEOUT = 20  # 20 seconds
-
+DEFAULT_RPC_BATCH_SIZE = 300
```

Comment:
Hm, actually, should we just make this 5000 (I think a 1MB block can hold about 4200 transactions), so we won't have to increase it if blocks start filling up more?